docs/HACKING.md

```diff
@@ -24,3 +24,11 @@ pip install --user -e .
 ```console
 python3.7 setup.py test
 ```
+# Check the Report for Unit Test Coverage
+```
+cd dffml
+coverage run -m unittest discover
```

Comment:
nit: Lets make this `coverage run setup.py test` as that will give people more verbose output (including downloading dependencies which adding `-v` to discover doesn't give) to see which tests are passing and failing by default.